usr/share/tb-profile-i2p/profile.i2p/user.js

```diff
@@ -0,0 +1,141 @@
+# Overrides for Extension Preferences
```

Comment:
Source of this file? Own creation? License?

Reply:
This was generated by firefox upon the first run, it combines the overrides into one profile. I'm going to remove it in my revision, as it will simpy be re-generated anyway.

Reply:
I was wrong. I generated this file by concatenating the two files in the preferences folder. This is because between versions 7.* and 8.* the preferences folder stopped working, *but* having both meant that the profile would work in both 7.* and 8.* while I was working on it. That makes user.js the more permanently neccessary.

Reply:
Could you please add the license of this file (and others) to https://github.com/Whonix/tb-starter/blob/master/debian/copyright? And then run `lintian --pedantic` (`make lintian`) to see if format is still ok?

Reply:
I'm not exactly sure of the mechanics of lintian on the copyright file, so I tried taking a look at the one in the Tor Debian package as an example. It did not appear to follow the format from the top of the copyright file in this repo, so I ran lintian --pedantic on it to see what it did, which didn't report an error or a warning. I added the Three-Clause BSD license to apply to usr/share/tb-profile-i2p/*, with a comment that they were sourced from the TBB. Hopefully that's OK?

Reply:
Made a build just now. debian/copyright and lintian looks ok.